CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/PeerToPeerRequestHandler.swift

```diff
@@ -85,7 +153,13 @@ public class PeerToPeerRequestHandler {
                 }
             }
             
-            let url = URL(string: "ws://\(host):\(port)/\(serverDBName)")!
+            if tls_disable != true {
+               wsPort = "wss"
+            } else {
+                wsPort = "ws"
+            }
```

Comment:
will this more readable?

if tls_disable {
   wsPort = "ws"
} else {
   wsPort = "wss"
}

Reply:
done